### PR TITLE
docs(concepts): describe virtual instances

### PR DESCRIPTION
**Pull request by eliobischof**
Describes the concept of virtual instances and their benefits

### CODE REVIEW

File: docs/docs/concepts/structure/instance.mdx [hunk 1]
```diff
@@ -2,8 +2,23 @@
...
 ---
 
-import InstanceDescription from './\_instance_description.mdx';
+## Instance Structure
 
-<InstanceDescription name="InstanceDescription" />
+An instance is the top node in ZITADEL's data hierarchy.
+Within an instance all the default [settings](/docs/concepts/structure/policies),
+such as branding, login policy, password policy, etc. for the system can be configured.
+One instance normally runs on one domain and has one issuer (e.g login.customer.com).
```
Comment (fforootd) [inline docs/docs/concepts/structure/instance.mdx:10]:
```suggestion
One instance normally runs on one domain and represents one issuer (e.g login.customer.com).
```

Comment (fforootd) [reply docs/docs/concepts/structure/instance.mdx:10]: ZITADEL is able to respond on multiple domains, but only one can be the main domain which is used to send mails and other things like the console access.

File: docs/docs/concepts/structure/instance.mdx [hunk 2]
```diff
@@ -2,8 +2,23 @@
...
 
-More about how to configure your instance read our [instance guide](../../guides/manage/console/instance-settings).
\ No newline at end of file
+One instance can contain multiple [organizations](/docs/concepts/structure/organizations),
+which in turn can represent the own company (e.g. departments) or business customers.
```
Comment (fforootd) [inline docs/docs/concepts/structure/instance.mdx:13]:
```suggestion
which in turn can represent your own company (e.g. departments), your business customers or a consumer organization.
```

File: docs/docs/concepts/structure/instance.mdx [hunk 3]
```diff
@@ -2,8 +2,23 @@
...
+
+Read more about how to configure your instance in our [instance guide](/docs/guides/manage/console/instance-settings).
+
+## Multiple Virtual Instances
+
+ZITADEL also has the concept of virtual instances.
+When installing ZITADEL from scratch, one instance is always automatically created for you.
+Nevertheless, you can add more virtual instances via the [system API](/docs/apis/proto/system#addinstance).
+This is especially useful if you have business customers, which in turn have their business customers with self service and custom domain demands.
```
Comment (fforootd) [inline docs/docs/concepts/structure/instance.mdx:22]:
```suggestion
This is useful if you have business customers, which in turn have their business customers with self service and custom domain demands.
```

File: docs/docs/concepts/structure/instance.mdx [hunk 4]
```diff
@@ -2,8 +2,23 @@
...
+By providing full-blown virtual ZITADEL instances, your customers have all the customization options available in ZITADEL.
```
Comment (fforootd) [inline docs/docs/concepts/structure/instance.mdx:23]:
```suggestion
By providing a virtual ZITADEL instances, your customers have all the customization options available in ZITADEL.
```

**Review by fforootd — COMMENTED** [review]
some small changes and comments

File: docs/docs/concepts/structure/instance.mdx (hunk 1 above)
Comment (fforootd) [reply docs/docs/concepts/structure/instance.mdx]: I think it is worth mentioning what I stated above

Comment (eliobischof) [reply docs/docs/concepts/structure/instance.mdx]: 😄 ok

Comment (fforootd) [reply docs/docs/concepts/structure/instance.mdx]: I thought so 😁

File: docs/docs/concepts/structure/_instance_description.mdx [hunk 5]
```diff
@@ -1,5 +0,0 @@
-An instance is the top hierarchy in the ZITADEL.
-Within an instance all the default [settings](/docs/concepts/structure/policies), such as branding, login policy, password policy, etc. for the system can be configured.
-One instance normally runs on one domain and has one issuer. (e.g login.customer.com)
-
-One instance can contain multiple [organizations](/docs/concepts/structure/organizations). Which can represent the own company or the customers.
```
Comment (hifabienne) [inline docs/docs/concepts/structure/_instance_description.mdx:5]: I think this is also used in another part. i think guides/console

Comment (eliobischof) [reply docs/docs/concepts/structure/_instance_description.mdx:5]: I can't find it
image

File: docs/docs/concepts/structure/instance.mdx [hunk 6]
```diff
@@ -2,8 +2,23 @@
...
-More about how to configure your instance read our [instance guide](../../guides/manage/console/instance-settings).
\ No newline at end of file
+One instance can contain multiple [organizations](/docs/concepts/structure/organizations),
+which in turn can represent your own company (e.g. departments), your business customers or a consumer organization.
+
+Read more about how to configure your instance in our [instance guide](/docs/guides/manage/console/instance-settings).
+
+## Multiple Virtual Instances
+
+ZITADEL also has the concept of virtual instances.
```
Comment (fforootd) [inline docs/docs/concepts/structure/instance.mdx:19]:
```suggestion
ZITADEL has the concept of virtual instances.
```

**Review by fforootd — DISMISSED** [review]
At least I have nothing more to add

**Review by hifabienne — APPROVED** [review]

**Review by fforootd — APPROVED** [review]

**Pull request merged**